spec/controllers/api/v1/bucketlists_controller_spec.rb

```diff
@@ -10,7 +10,7 @@
   it_behaves_like "an api controller", id: "not-existing"
 
   describe 'GET #index' do
-    before { get :index }
+    let!(:req) { get :index }
```

Comment:
**Test Functionality - Critical**
Remove before hooks for unnecessary overhead. `let!` would only be called once for all examples.